README.md

```diff
@@ -140,7 +140,23 @@ Example:
 
 ```
 
+### variable replacement
+The `gp-translate` directive supports an optional `values` attribute that allows passing of custom data to it.  This is used to inject
+variables into the returned translation string.
 
+Example translation string:
+
+`"DATA-KEY-WITH-VARIABLE" : "Welcome {{user}}"`
+
+Example usage:
+
+```html
+
+    <gp-translate key="DATA-KEY-WITH-VARIABLE" bundle="someBundle" values="{{controllerAsVariable.userName}}"/>

```

Comment:
`{{controllerAsVariable.userName}}` would get substituted/interpolated prior to being processed by the directive.  The only requirement is that whatever is in there is valid JSON.

In my testing, the controller had a variable declared like this:

`userName = { user : "John Doe" };`

Any suggestion on improving the documentation to make it more clear?